### ISSUE TITLE
Error accessing methods in www before the full start

### ISSUE THREAD

**Issue by belochub** [metarhia/Impress#722]
Stack trace: 
```
2017-04-14T11:47:50.708Z	Worker(4073/S1N1)	[impress]
TypeError: application.log.access is not a function;
Client.accessLog (impress.client.js:107:21);
impress.client.js:635:12;
Client.saveSession (impress.client.js:403:10);
Client.end (impress.client.js:612:10);
impress.index.js:73:24;
impress.templating.js:204:7;
next (/metasync/metasync.js:372:17);
/metasync/metasync.js:376:14;
impress.templating.js:185:9;
next (/metasync/metasync.js:373:12);
/metasync/metasync.js:376:14;
...
Client.render (impress.templating.js:181:18);
impress.templating.js:188:16;
next (/metasync/metasync.js:373:12);
/metasync/metasync.js:376:14;
impress.templating.js:185:9;
next (/metasync/metasync.js:373:12);
Object.metasync.series (/metasync/metasync.js:380:3);
Client.render (impress.templating.js:181:18);
impress.templating.js:102:16;
tryToString (fs.js:455:3);
FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:442:12)
```
`impress version` output:
```
Impress Application Server 0.3.36
 Impress AS: 0.3.36
    Node.js: 6.10.0
         v8: 5.1.281.93
      libuv: 1.9.1
       zlib: 1.2.8
   Open SSL: 1.0.2k
HTTP parser: 2.7.0
         OS: Linux 3.10.0-327.18.2.el7.x86_64 x64
```

**Comment by tshemsedinov** [comment]
We will reply with HTTP code `503 Service Unavailable`

**Comment by belochub** [comment]
@tshemsedinov, I created similar issue for JSTP connections, take a look please https://github.com/metarhia/Impress/issues/726.

**Comment by tshemsedinov** [comment]
@belochub @aqrln here is patch https://github.com/metarhia/Impress/pull/724 for `0.3.x`

**Comment by tshemsedinov** [comment]
Fixed for both active branches `v0.3` and `v0.4`
